app/models/bookshelf.rb

```diff
@@ -0,0 +1,15 @@
+require_dependency 'dpla/items'
+##
+# A model that represents a Book item retrieved from the search.
+#
+# For displaying the books in BookshelfController.
+# Search#conditions is overridden to forego the retrieval of facets.
+#
+# @see BookshelfController
+#
+class Bookshelf < Search
+
+  def conditions
+    { q: @term, facets: [], facet_size: 0 }.merge(@filters).merge(@args)
```

Comment:
That's interesting, because I was getting a different result.  Are you sure that you don't have the queries cached?  I was flushing memcache and the proxy cache.  It appeared that `facets` and `facet_size` were getting default values without being overridden explicitly, but I could be wrong.  I could have been the one looking at something that was cached.